linux_vmstat.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+#
+# igcollect - Linux virtual memory stats
+#
+# Copyright (c) 2016, InnoGames GmbH
+#
+
+from __future__ import print_function
+from socket import gethostname
+import time, sys
+
+# utility to read and parse a space delimited file (meminfo)
+def parse_split_file(filename):
+    try:
+        with open(filename, 'r') as f:
+            return [line.strip().split(None, 1) for line in f]
+    except:
```

Comment:
What about when one has no read access for the file. Will be raise an error then? Maybe we should just let it be raised.